YOUR_ADMIN/includes/init_includes/init_bc_config.php

```diff
@@ -61,110 +61,122 @@
+            ('<b>Header Nav Bar Link Color</b>', 'ZCA_HEADER_NAVBAR_LINK_COLOR', '#ffffff', 'Default:#ffffff', " . $bccid . ", 43, now()),
+            ('<b>Header Nav Bar Link Color on Hover</b>', 'ZCA_HEADER_NAVBAR_LINK_COLOR_HOVER', '#cccccc', 'Default:#cccccc', " . $bccid . ", 44, now()),
+            ('<b>Header Nav Bar Button Text Color</b>', 'ZCA_HEADER_NAVBAR_BUTTON_TEXT_COLOR', '#ffffff', 'Default:#ffffff', " . $bccid . ", 45, now()),
+            ('<b>Header Nav Bar Button Text Color on Hover</b>', 'ZCA_HEADER_NAVBAR_BUTTON_TEXT_COLOR_HOVER', '#cccccc', 'Default:#cccccc', " . $bccid . ", 46, now()),
+            ('<b>Header Nav Bar Button Color</b>', 'ZCA_HEADER_NAVBAR_BUTTON_COLOR', '#343a40', 'Default:#343a40', " . $bccid . ", 47, now()),
+            ('<b>Header Nav Bar Button Color on Hover</b>', 'ZCA_HEADER_NAVBAR_BUTTON_COLOR_HOVER', '#919aa1', 'Default:#919aa1', " . $bccid . ", 48, now()),
+            ('<b>Header Nav Bar Button Border Color</b>', 'ZCA_HEADER_NAVBAR_BUTTON_BORDER_COLOR', '#343a40', 'Default:#343a40', " . $bccid . ", 49, now()),
+            ('<b>Header Nav Bar Border Color on Hover</b>', 'ZCA_HEADER_NAVBAR_BUTTON_BORDER_COLOR_HOVER', '#919aa1', 'Default:#919aa1', " . $bccid . ", 50, now()),
+            ('Header Category Tabs Text Color', 'ZCA_HEADER_TABS_TEXT_COLOR', '#ffffff', 'Default:#ffffff', " . $bccid . ", 51, now()),
+            ('Header Category Tabs Text Color on Hover', 'ZCA_HEADER_TABS_TEXT_COLOR_HOVER', '#007faf', 'Default:#007af', " . $bccid . ", 52, now()),
+            ('Header Category Tabs Border Color on Hover', 'ZCA_HEADER_TABS_BORDER_COLOR_HOVER', '#007faf', 'Default:#007faf', " . $bccid . ", 53, now()),
+            ('Header Category Tabs Background Color', 'ZCA_HEADER_TABS_BACKGROUND_COLOR', '#007faf', 'Default:#007faf', " . $bccid . ", 54, now()),
+            ('Header Category Tabs Background Color on Hover', 'ZCA_HEADER_TABS_BACKGROUND_COLOR_HOVER', '#ffffff', 'Default:#ffffff', " . $bccid . ", 55, now()),
+            ('Header Category Tabs Active Color', 'ZCA_HEADER_TABS_ACTIVE_COLOR', '#ad0000', 'Default:#ad0000', " . $bccid . ", 56, now()),
+            ('Header Category Tabs Active Background Color', 'ZCA_HEADER_TABS_ACTIVE_BACKGROUND_COLOR', '#ffffff', 'Default:#ffffff', " . $bccid . ", 57, now()),
+            ('Header Category Tabs Active Border Color', 'ZCA_HEADER_TABS_ACTIVE_BORDER_COLOR', '#ad0000', 'Default:#ad0000', " . $bccid . ", 58, now()),
+            ('<b>Header EZ-Page Bar Background Color</b>', 'ZCA_HEADER_EZPAGE_BACKGROUND_COLOR', '#464646', 'Default:#464646', " . $bccid . ", 59, now()),
+            ('<b>Header EZ-Page Bar Link Color</b>', 'ZCA_HEADER_EZPAGE_LINK_COLOR', '#ffffff', 'Default:#ffffff', " . $bccid . ", 60, now()),
+            ('<b>Header EZ-Page Bar Link Color on Hover</b>', 'ZCA_HEADER_EZPAGE_LINK_COLOR_HOVER', '#464646', 'Default:#464646', " . $bccid . ", 61, now()),
```

Comment:
Noting that this color update makes the EZ-Page link on hover pretty much invisible.  Should I revert to the `#cccccc` that's used in the footer?